packages/manager/src/features/Billing/PdfGenerator/PdfGenerator.ts

```diff
@@ -164,13 +165,25 @@ const addTitle = (doc: jsPDF, y: number, ...textStrings: Title[]) => {
   doc.setFont(baseFont, 'normal');
 };
 
-export const printInvoice = (
+// M3-6177 only make one request to get the logo
+const getAkamaiLogo = () => {
+  return axios
+    .get(AkamaiLogo, { responseType: 'blob' })
+    .then((res) => {
+      return URL.createObjectURL(res.data);
+    })
+    .catch(() => {
+      return AkamaiLogo;
+    });
+};
```

Comment:
I think the same thing could be done with fetch, which removes the dependency on axios. But this is optional because the entire repo basically depends on Axios

```suggestion
const getAkamaiLogo = () => {
  return fetch(AkamaiLogo)
    .then(async (res) => {
      const blob = await res.blob();
      return URL.createObjectURL(blob);
    })
    .catch(() => {
      return AkamaiLogo;
    });
};
```